apps/src/panels/types.ts

```diff
@@ -15,7 +15,10 @@ export type PanelLayout =
 export interface Panel {
   imageUrl: string;
   text: string;
+  key: string;
   nextUrl?: string;
   layout?: PanelLayout;
-  key: string;
+  dark?: boolean;
```

Comment:
Also minor: mind leaving a comment that these fields are not in levelbuilder yet (ex. because they're experimental and may be updated soon?)

Reply:
Good call.  Done.